README.md

```diff
@@ -1,3 +1,30 @@
-# RPI-Security-Sensor
-
+# Security Sensor
 ## Makers October 2019 Cohort Final Project
+
+A home security system, using a Raspberry Pi, motion sensor, and camera. Upon detecting motion, a picture is taken and sent to a hosted website. A notification is also sent to your phone, including a link to the picture URL.
+
+### Getting Started
```

Comment:
Start these lines with uppercase as well

Reply:
none of these lines need to be changed - this is a paragraph, not bullet points